osu.Game/Screens/Play/Player.cs

```diff
@@ -76,6 +76,15 @@ public class Player : ScreenWithBeatmapBackground
 
         protected GameplayClockContainer GameplayClockContainer { get; private set; }
 
+        protected StoryboardContainer StoryboardContainer { get; private set; }
+
+        protected virtual StoryboardContainer CreateStoryboardContainer(Storyboard storyboard) => new StoryboardContainer(storyboard)
+        {
+            RelativeSizeAxes = Axes.Both,
+            Alpha = 1,
```

Comment:
are these defaults actually required? if so, why?

Reply:
The default alpha doesn't seem to be necessary, but the user dim default is there since there's never a time outside of Player where we might need to disable it, unlike Backgrounds.

Now that I think about it, could probably move this inside `StoryboardContainer`